content/change/institutions/lab.md

```diff
@@ -0,0 +1,17 @@
+title: lab
+
+# How can I convince my lab to implement open science practices?
+
+* **Workshops** are often a great way to get your fellow researchers in touch with new methods.
+** If you are up to it yourself, you might find inspiration in the [Open Science Workshop Materials of the LMU Open Science Center](https://osf.io/zjrhu/).
+** The COS provides an Open and Reproducible Research Trainer Space which contains a vast body of material for Open Science workshops including sections on workshop planning (pre workshop checklists, flyers, email tempates, etc.) and curriculum (slides, full manual).
+** Or get in touch with one of the speakers on [this list](https://osf.io/tbkzh/wiki/Speakers%20for%20talks%20and%20workshops%20on%20Open%20Science%20topics/), who cover several topics in the field of Open Science.
+* If you're in one of those labs, where people are indifferent about replicability or even openly take a stand against it, you might find some arguments for your next discussion in one of these resources:
+** [Zwaan, R., Etz, A., Lucas, R. E., & Donnellan, B. (2018, August 1). Making Replication Mainstream.](https://doi.org/10.31234/osf.io/4tg9c)
+** [Open Science Talking Points](https://docs.google.com/document/d/1CJFdbxD9Z7xCWpXPW8XIKKpoXK-TGk9_SsVIqw9FhiM/edit), assembled at the SIPS Meeting 2018
+
+# What should I think of, when I want to transition my lab to open science practices?
+
+The best piece of advice is probably to take it step by step and don't put too much pressure on it. Research is a continuous process, so you can try to implement new practices along the way. (To make that a bit easier, we tried to structure this ressource based on the scientific circle.)
+
+* Some people at the SIPS Meeting 2018 put together some [links to useful resources](https://osf.io/f529n/wiki/home/ ) for transitioning to open science practices in labs.
```

Comment:
I think there are some great resources in this collection, but they look fairly generic to me -- I'm not sure they are particularly suited to this particular question.

Reply:
Can we use some of the links elsewhere?